README.md

```diff
@@ -18,7 +20,7 @@ TEST_CACHE_NAME="test-cache" # Cache name
 
 To set up the integration tests, you'll need two Docker containers: one for running Redis and the other for running the integration tests.
```

Comment:
We will probably have to remove all of this and direct them to use devcontainers, right? or maybe add another section like a second way to develop and run the examples?

Reply:
Yeah. That, or move it to a CONTRIBUTING.md file. I can make this pr more general if desired, but I limited it to language updates as a first pass.